wee_slack.py

```diff
@@ -3311,6 +3314,39 @@ def linkify_word(match):
     return re.sub(linkify_regex, linkify_word, message_escaped, re.UNICODE)
 
 
+def unfurl_blocks(message_json):
+    block_text = []
+    for block in message_json["blocks"]:
+        try:
+            if block["type"] == "section":
+                if "text" in block: block_text += unfurl_texts([block["text"]])
+                if "fields" in block: block_text += unfurl_texts(block["fields"])
+            elif block["type"] == "actions":
+                block_text.append("|".join(i["text"]["text"] for i in block["elements"]))
+            elif block["type"] == "call":
+                block_text.append(". Join via ").append(block["call"]["v1"]["join_url"])
+            elif block["type"] == "divider":
+                block_text.append("\n")
+            elif block["type"] == "context":
+                block_text.append("|".join(i["text"] for i in block["elements"]))
+            else:
+                raise NotImplementedError("Block type not implemented", block["type"])
```

Comment:
Instead of raising an exception here, I think we should render `<Unsupported block type "type">` in the message, and log the block to the debug buffer.

Reply:
OK, I added a solution for this.  I use the same coloring as for reactions.  I'm not sure if this is the best solution.

Reply:
Hm, don't think we should reuse the color for reactions. But it feels a bit unnecessary to add a separate config for it, since it hopefully won't be shown that often.

I added a config for deleted messages now, `color_deleted`. What do you think about using that?

Reply:
Sounds good.